README.md

```diff
@@ -20,22 +20,48 @@ Go to the official [Code For Life website][c4l].
 
 ## To run the app locally
 
-* Install prerequisites. E.g. on Ubuntu / Linux Mint:
+* Clone the repo: `https://github.com/ocadotechnology/rapid-router.git`. Fork it first if you want to contribute, and work on a separate branch for your work.
+
+### Ubuntu / Linux Mint
+* Install prerequisites
     * `sudo apt-get install git`
     * `sudo apt-get install python-dev python-pip`
     * `sudo pip install virtualenvwrapper`
     * `sudo apt-get install libxml2-dev libxslt1-dev zlib1g-dev gettext`
     * `sudo apt-get install ruby2.0` - still Ruby 1.9 hiding under `ruby` command.
     * `sudo gem install sass -v 3.3.4` - later versions incompatible with Ruby 1.9 (see above).
 
-* Clone the repo: `https://github.com/ocadotechnology/rapid-router.git`. Fork it first if you want to contribute, and work on a separate branch for your work.
 * Make and activate a virtualenv
     * e.g. the first time, `mkvirtualenv -a path/to/rapid-router rapid-router`
     * and thereafter `workon rapid-router`
     * **NOTE:** If you have trouble running these commands (e.g. `Command not found: workon`), refer to [this article][virtualenv], the top answer should help you resolve them. You may need to run:
 
           export WORKON_HOME=~/Envs
           source /usr/local/bin/virtualenvwrapper.sh
+
+### Mac OS
+* Install prerequisites
+    * `brew install rbenv`
+    * `brew install pyenv`
+    * `brew install pyenv-virtualenv`
+
+* Update `.zshrc` with the following
```

Comment:
Is it possible to have instructions for people using a .bash_profile? (or to make it clear that this could be added in that also